### PR TITLE
CMOS-460 add cbhealthagent dl links

### PR DESCRIPTION
**Pull request by aemery-cb**
updated docs to contains cbhealth agent dl links
also included a change to make sure we're getting all the issues from the release docs generation.

### CODE REVIEW

File: docs/modules/ROOT/pages/health-agent.adoc [hunk 1]
```diff
@@ -13,7 +13,8 @@ However, it can also assist in setting up a complete observability stack by prov
 
 Download the package:
 
-// TODO
+- https://packages.couchbase.com/couchbase-observability-stack/dp-2/cbhealthagent-linux-amd64-dp2.deb[`cbhealthagent-linux-amd64.deb`^]
+- https://packages.couchbase.com/couchbase-observability-stack/dp-2/cbhealthagent-linux-amd64-dp2.rpm[`cbhealthagent-linux-adm64.rpm`^]
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/health-agent.adoc:17]: You have `dp-2` in one place in the URL, and `dp2` in another - that doesn't seem right.

**Review by markspolakovs — CHANGES_REQUESTED** [review]
It might be better to wait until the release is done and we have the final URLs, rather than guessing - we can update the docs after the fact.

The embedded/built copy would then go out of date, but we could put in a "check the Couchbase website" reference to the page where the final links will be.

File: docs/modules/ROOT/pages/health-agent.adoc (hunk 1 above)
Comment (markspolakovs) [inline docs/modules/ROOT/pages/health-agent.adoc:17]:
```suggestion
- https://packages.couchbase.com/couchbase-observability-stack/dp-2/cbhealthagent-linux-amd64-dp2.rpm[`cbhealthagent-linux-amd64.rpm`^]
```

Comment (aemery-cb) [reply docs/modules/ROOT/pages/health-agent.adoc:17]: how did i mess up copying and pasting?!?

**Pull request merged**